--- lib/postgres/11/Dockerfile	2019-06-17 22:17:26.847311579 +0900
+++ lib/postgres/11/Dockerfile.modifided	2019-06-17 22:24:43.532973109 +0900
@@ -166,7 +166,7 @@ ENV PATH $PATH:/usr/lib/postgresql/$PG_M
 ENV PGDATA /var/lib/postgresql/data
 # this 777 will be replaced by 700 at runtime (allows semi-arbitrary "--user" values)
 RUN mkdir -p "$PGDATA" && chown -R postgres:postgres "$PGDATA" && chmod 777 "$PGDATA"
-VOLUME /var/lib/postgresql/data
+#VOLUME /var/lib/postgresql/data
 
 COPY docker-entrypoint.sh /usr/local/bin/
 RUN ln -s usr/local/bin/docker-entrypoint.sh / # backwards compat
